test/test_utils.c

```diff
@@ -295,17 +307,29 @@ int _test_nx_inflate(Byte* compr, unsigned int comprLen, Byte* uncompr,
 	if (time != NULL)
 		gettimeofday(&time->start, NULL);
 
-	while (d_stream.total_out < uncomprLen) {
+	d_stream.avail_out = step;
+	do {
 		if (d_stream.total_in < comprLen)
 			d_stream.avail_in = step;
-		d_stream.avail_out = step;
-		err = nx_inflate(&d_stream, flush);
+		err = nx_inflate(&d_stream, Z_NO_FLUSH);
 		if (err == Z_STREAM_END) break;
+		/* Z_BUF_ERROR could mean that we need more output. */
+		if (err == Z_BUF_ERROR) {
+			/* Get more room for output */
+			d_stream.avail_out += step;
```

Comment:
The test asked a function to decompress at a specific step. It looks wrong to use a different step than the one requested by the test.
In other words, if the decompression doesn't work either the test is wrong or the decompression function is broken.

There is a chance this test won't be able to use this function.

Reply:
After other changes I re-tested it and this is not needed anymore. Thanks.